site/README.md

```diff
@@ -1,18 +0,0 @@
-# IPFS Distributions
-
-
-## Development
-
-Harpjs setup with
-
-* Live reload using browser-sync
-* JavaScript
-  * bundeling using webpack
-  * transpiling using babel
-* Linting using eslint
-
-
-```bash
-$ gulp
-$ open localhost:3000
-```
```

Comment:
i think it made perfect sense to keep this here. very few people using this repo will need to touch the site. they _should not_ have to know any of this or worry about it.